wandb/apis/public.py

```diff
@@ -2519,32 +2619,88 @@ def copy(cache_path, target_path):
                     util.mkdir_exists_ok(os.path.dirname(target_path))
                     # We use copy2, which preserves file metadata including modified
                     # time (which we use above to check whether we should do the copy).
-                    shutil.copy2(cache_path, target_path)
+                    if os.path.islink(cache_path):
```

Comment:
Does this work if you've already downloaded the artifact to whatever directory? Or does it throw an error?

Reply:
This works on successive downloads. But i did add a conditional measure to safely relink in the case of a duplicate download in next commit